test/e2e/crd_e2e_test.go

```diff
@@ -304,8 +307,21 @@ var _ = Describe("CRD Versions", func() {
 
 		// update channel on sub
 		sub.Spec.Channel = alphaChannel
-		_, err = crc.OperatorsV1alpha1().Subscriptions(testNamespace).Update(context.TODO(), sub, metav1.UpdateOptions{})
-		Expect(err).ToNot(HaveOccurred(), "could not update subscription")
+		sub.TypeMeta.APIVersion = "operators.coreos.com/v1alpha1"
+		sub.TypeMeta.Kind = "Subscription"
+		sub.ManagedFields = []metav1.ManagedFieldsEntry{}
+
+		// use server-side apply to apply the update to the subscription point to the alpha channel
+		forceApply := true
+		opts := metav1.PatchOptions{
+			Force:        &forceApply,
```

Comment:
The apply should only need to be forced if anyone else is writing into the spec, right? Which in this test we know isn't happening.

Reply:
```
              Message: "Apply failed with 1 conflict: conflict with \"e2e.test\" using operators.coreos.com/v1alpha1: .spec.channel",
```
Looks like force is required?

Reply:
that makes sense.  I think if we used apply to create the subscription in the first place, then it would succeed, but this works.